docs/guides/custom-redirects.mdx

```diff
@@ -219,15 +231,103 @@ export default function Home() {
     <div>
       <h1>Welcome!</h1>
 
-      <SignInButton signInFallbackRedirectUrl="/dashboard" signUpFallbackRedirectUrl="/onboarding">
+      <SignInButton fallbackRedirectUrl="/dashboard" signUpFallbackRedirectUrl="/onboarding">
         Sign in
       </SignInButton>
 
-      <SignUpButton signInFallbackRedirectUrl="/dashboard" signUpFallbackRedirectUrl="/onboarding">
+      <SignUpButton signInFallbackRedirectUrl="/dashboard" fallbackRedirectUrl="/onboarding">
         Sign up
       </SignUpButton>
     </div>
   );
 }
 ```
 </CodeBlockTabs>
+
+### UI Components
+
+[`<SignIn>`](/docs/components/authentication/sign-in) and [`<SignUp>`](/docs/components/authentication/sign-up) also take these properties.
+
+<Callout type="info">
+It is recommended to define both `signInFallbackRedirectUrl` and `signUpFallbackRedirectUrl` redirects in each component as some users may choose to sign up instead after attempting to sign in.
+</Callout>
+
+<Tabs type="framework" items={["Next.js", "React", "Remix", "Gatsby"]}>
+  <Tab>
+    ```tsx filename="app/page.tsx"
+    import { SignIn, SignUp, useUser } from "@clerk/nextjs";
+
+    export default function Home() {
+      const { user } = useUser();
+
+      if (!user) {
+        return (
+          <>
+            <h1>Sign in or sign up</h1>
+            <div>
+              <SignIn fallbackRedirectUrl="/dashboard" signUpFallbackRedirectUrl="/onboarding" />
```

Comment:
here again,  I don't see a `fallbackRedirectUrl` defined on this page - only `signInFallbackRedirectUrl` and `signUpFallbackRedirectUrl`